zgw_consumers/mixins.py

```diff
@@ -0,0 +1,80 @@
+"""
+Implement :class:`django.db.models.FileField` related utilities.
+
+These utilities apply to file fields and subclasses thereof.
+"""
```

Comment:
The License of Open Forms (EUPL-1.2) should be included/mentioned here as this code is lifted from there. See https://github.com/open-formulieren/open-forms/blob/master/LICENSE.md - "5. Obligations of the Licensee". I'm not sure about the specifics of all this as we are the original authors, but we can check with some people (leaving this comment as a self-reminder).